backend/gn_module_monitoring/migrations/e2b66850b5ee_add_digitiser_to_t_sites_groups.py

```diff
@@ -0,0 +1,51 @@
+"""Add digitiser to t_sites_groups
+
+Revision ID: e2b66850b5ee
+Revises: 6a15625a0f4a
+Create Date: 2023-09-11 12:17:17.280948
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "e2b66850b5ee"
+down_revision = "6a15625a0f4a"
+branch_labels = None
+depends_on = None
+
+monitorings_schema = "gn_monitoring"
+table = "t_sites_groups"
+column = "id_digitiser"
+
+foreign_schema = "utilisateurs"
+table_foreign = "t_roles"
+foreign_key = "id_role"
+
+
+def upgrade():
+    op.add_column(
+        table,
+        sa.Column(
+            column,
+            sa.Integer(),
+            sa.ForeignKey(
+                f"{foreign_schema}.{table_foreign}.{foreign_key}",
+                name=f"fk_{table}_{column}",
+                onupdate="CASCADE",
+            ),
+            nullable=False,
```

Comment:
la colonne ne peut avoir la contrainte NOT NULL car il y a des données saisies et il n'est pas possible à 100% de retrouver l'id_digitizer